src/javafundamentals/interfaces/Ex01/Ex01Interfaces.java

```diff
@@ -0,0 +1,25 @@
+package javafundamentals.interfaces.Ex01;
+
+
+/**
+ * Create a couple of classes that have some behaviour and some specific behaviour in common
```

Comment:
You are missing the part of the specific behaviour of the classes. So some methods in the SellableHouse and the SellableCar that are specific for them and they are not in the interface.

Reply:
I added the specific class methods that aren't included in the interface!